config/security.java.policy-server

```diff
@@ -131,6 +131,7 @@ grant principal org.ow2.proactive.authentication.principals.GroupNamePrincipal "
 	//"false" means user can get full state and listen for any events.
     permission org.ow2.proactive.scheduler.permissions.HandleOnlyMyJobsPermission "true";
     permission org.ow2.proactive.scheduler.permissions.ChangePriorityPermission "1,2,3";
+    permission org.ow2.proactive.scheduler.permissions.HandleJobsWithGenericInformationPermission "";    
```

Comment:
What if the permission is not defined at all in the policy file, will it produce an error (i.e. backward compatibility) ?